lib/fathead/beautifulsoup/parse.py

```diff
@@ -78,21 +72,27 @@ def parse_first_paragraph_from_data(self, section):
         Get the first paragraph for display
         Args:
             section: A section of parsed HTML
-
         Returns:
             First paragraph in the HTML
         """
-        data = section.find('p')
+        data = section.find_all('p')
         if data:
-            return data.text.replace('\n', ' ').replace('  ', ' ')
+            arg = ''
+            for index,d in enumerate(data):
+                data[index] = d.text.replace('\n', ' ').replace('  ', ' ')
```

Comment:
Looks like we need to modify https://github.com/duckduckgo/zeroclickinfo-fathead/pull/439/files#diff-94d0e0de1b7b7081617a1b3e8369988fR101 to also have `.replace('\\x', '\\\\x')